manifests/params.pp

```diff
@@ -2,69 +2,204 @@
 #
 # Sets varables for the SSH class
 #
-class ssh::params {
```

Comment:
This file is ridiculous. I decided not to pull it into this PR, but we should switch to hiera (https://github.com/ploperations/ploperations-ssh/issues/9).